styles.css

```diff
@@ -469,3 +469,108 @@ h1 {
   border: solid 1px;
   background-color: white;
 }
+
+.contact-form {
+  padding: 114px 24px;
+  background-image: url(./images/contact\ form\ shapes-mobile@2x.png);
+  min-height: 800px;
+  background-repeat: no-repeat;
+  background-size: 63% 83%;
+  background-position: right 30px;
+  border-top-left-radius: 5rem;
+  background-color: #6070ff;
+  margin-top: -4rem;
+  display: grid;
+  grid-template-rows: auto auto;
+}
```

Comment:
- [OPTIONAL] Good job adding the background image in the form section as suggested by the code reviewer, also good job using `Display: grid;` to layout components. however, I believe the requirement here to use `Display: flex;` to layout form elements. Therefore I will highly recommend you to apply flex instead of the grid here.